MaxText/decode.py

```diff
@@ -250,12 +251,12 @@ def decode_loop(config, state=None):
     f"generated text mismatch {new_text=} {config.autoregressive_decode_assert=}"
 
   num_steps = len(steps)
-  elapsed_time = (endtime-starttime).total_seconds()
+  elapsed_time = (endtime-starttime).total_seconds() * 1000
```

Comment:
I'm actually going to delete this code soonish so you don't need to sweat it ... but this var should be named `elapsed_time_ms` now (and I should have named it `elapsed_time_seconds`)